### PR TITLE
complete js-variables-and-numbers_social-media-post challenge

### PR DESCRIPTION
**Pull request by spacedoe**
# Context

This PR aims to tackle [The social media post challenge](https://github.com/spiced-academy/honey-web-dev/blob/js-variables-and-numbers/sessions/js-variables-and-numbers/challenges-js-variables-and-numbers.md#data-for-a-social-media-post).

### CODE REVIEW

File: javascript/js-variables-and-numbers/js-variables-and-numbers_social-media-post/js/index.js
Comment (maria-tsek) [inline javascript/js-variables-and-numbers/js-variables-and-numbers_social-media-post/js/index.js:1]: It's looks good!

**Review by maria-tsek — COMMENTED** [review]
good job!

**Review by cincu — COMMENTED** [review]
good content :)

File: javascript/js-variables-and-numbers/js-variables-and-numbers_social-media-post/js/index.js
Comment (spacedoe) [reply javascript/js-variables-and-numbers/js-variables-and-numbers_social-media-post/js/index.js:1]: Thx!

**Pull request merged**